workshops/github-copilot/workshop.md

```diff
@@ -851,96 +850,6 @@ You must now develop and add new features to your server. The requests that the
 |**/MakeZipFile**|Using zlib create a zip file called `sample.gz` that contains `sample.txt`|
 |**/RandomEuropeanCountry**|Make an array of european countries and its ISO codes<br/>Return a random country from the array<br/>Return the country and its ISO code|
 
-## GitHub Copilot Labs exercises
-
-These tasks can be performed with the [GitHub Copilot Labs](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-labs) add-in, currently PREVIEW functionality, expect some bugs or weird behavior (you've been warned <:o)).
```

Comment:
I’ve removed this section, but would it be more appropriate to adapt these exercises for use with GitHub Copilot Chat?

Reply:
This could be a good idea if you have the time to do the update

Reply:
Thanks! I'll try that in a coming weeks.

Reply:
Hi @sinedied, I've made some changes in https://github.com/microsoft/moaw/pull/107/commits/646580cf23aacec07764bb8a324d5ec62d2ff968 so that you can use GitHub Copoilot Chat instead of GitHub Copilot Labs here. Some functionalities still don't work well on GitHub Copilot Chat, so I've removed those sections. Could you review again?